### DISCUSSION TITLE
Win7重启系统，会卡在关闭桌面的那个画面，一直转圈。

### DISCUSSION THREAD

**Discussion by CraftX2025** [natfrp/wiki#172] [使用问题]
Win7下使用没有问题，但是在重启系统的时候，无法正常重启，会卡在结束桌面的那个地方，需要按reset重启。
如果手动退出软件，重启系统没有问题，这几年都是如此，win10倒是没问题，正常重启。

**Comment by HPPC114514** [comment]
那就重启前关掉软件啊 _（不是_

**Reply by CraftX2025** [reply]
不是每次都能记得，升级win10了，问题已解决。
